scripts/generate_release_updates.py

```diff
@@ -0,0 +1,137 @@
+    '   %s.\n'
+    '   %s\n'
+    '   Finally, I\'d like to thank %s for their help with pre-release '
+    'testing, bug-fixing and QA, as well as %s for leading the QA team for '
+    'this release.\n\n'
+    'Thanks,\n'
+    '%s\n') % tuple(SECTIONS_TO_ADD)
+
+PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+RELEASE_MAIL_MESSAGE_FILEPATH = os.path.join(
+    PARENT_DIR, 'release_mail_message.txt')
+
+
+def create_new_file_with_release_message_template():
+    """Adds the template message to release mail filepath."""
+    with python_utils.open_file(RELEASE_MAIL_MESSAGE_FILEPATH, 'w') as f:
+        f.write(RELEASE_MAIL_MESSAGE_TEMPLATE)
+
+    common.ask_user_to_confirm(
+        'Please make updates to following file %s for generating the '
+        'release announcement mail by adding:\n'
+        '   1. Main changes for release\n'
+        '   2. Editorials/announcements if required\n'
+        '   3. Author details from release_summary.md\n'
+        '   4. Names of release testers\n'
+        '   5. Name of QA Team lead\n'
+        '   6. Your name\n' % RELEASE_MAIL_MESSAGE_FILEPATH)
+
+
+def validate_release_message():
+    """Checks the message after the mail template is updated by the
+    user.
+
+    Raises:
+        Exception: The message still contains sections from template
+            which are not updated.
+    """
+    with python_utils.open_file(RELEASE_MAIL_MESSAGE_FILEPATH, 'r') as f:
+        release_mail_message = f.read()
+        extra_sections = [
+            section for section in SECTIONS_TO_ADD if (
+                section in release_mail_message)]
+        if extra_sections:
+            raise Exception(
+                'Template not formatted correctly. '
+                'Following sections still not updated: %s.\n'
+                'Please re-run the scripts and make the updates again.' % (
+                    ', '.join(extra_sections)))
+
+
+def prompt_user_to_send_announcement_email():
+    """Asks the user to send release announcement mail and check if
+    it is in announcement category.
+    """
+    common.open_new_tab_in_browser_if_possible(
+        'https://www.gmail.com')
+    common.ask_user_to_confirm(
+        'Please copy the mail message from %s and send the email to:\n'
+        '   TO: oppia-dev@googlegroups.com\n'
+        '   BCC: oppia@googlegroups.com\n'
+        '   BCC: oppia-announce@googlegroups.com\n'
+        '   BCC: each new contributor for this release\n' % (
+            RELEASE_MAIL_MESSAGE_FILEPATH))
+
+    common.open_new_tab_in_browser_if_possible(
+        'https://groups.google.com/forum/#!categories/oppia/announcements')
+    common.ask_user_to_confirm(
+        'Ensure the email sent to oppia@ is in the Announcements category')
+
+
+def main():
+    """Performs task to generate message for release announcement."""
+    branch_name = common.get_current_branch_name()
+    if not re.match(r'release-\d+\.\d+\.\d+$', branch_name):
```

Comment:
should we move this regex to a constant as it's used in multiple places/files?

Reply:
Moved this to `common.is_current_branch_a_release_branch()` instead of having different checks everywhere.